raiden/tests/integration/network/proxies/test_token_network_registry.py

```diff
@@ -87,6 +91,7 @@ def test_token_network_registry(
             token_address=test_token_address,
             channel_participant_deposit_limit=TokenAmount(UINT256_MAX),
             token_network_deposit_limit=TokenAmount(UINT256_MAX),
+            block_identifier=preblockhash,
```

Comment:
@rakanalh If I say "latest" here, I get a different kind of exception (`BrokenPreconditionError`).